ajax/libs/videojs-thumbnails/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "videojs-thumbnails",
+  "description": "progress bar thumbnails plugin for video.js",
+  "version": "0.1.1",
+  "author": "Chris Tierney <ctierney@brightcove.com>",
```

Comment:
Seems like not the author. He didn't contribute to this project according to https://github.com/brightcove/videojs-thumbnails/graphs/contributors